### PR TITLE
update readme with installation guide

### PR DESCRIPTION
**Pull request by ebru-backbase**
Adding installation guide for dummies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,18 @@ bb-fuel supports multiple DBS versions. See below which version maps to the requ
 Note: Starting version 2.4.0 Kubernetes is added as supported platform.
 Note: Starting version 2.0 PCF is added as supported platform.
 
+## Installation
+
+- Install [IntelliJ](https://www.jetbrains.com/idea/) IDE.
```
Comment (lehphyro) [inline README.md:144]: Intellij is not required.

File: README.md [hunk 2]
```diff
@@ -139,6 +139,18 @@ bb-fuel supports multiple DBS versions. See below which version maps to the requ
...
+- [Java](https://www.java.com/en/) installed
+- Install and configure [Maven](https://www.jetbrains.com/help/idea/maven-support.html#maven_import_project_start).
```
Comment (lehphyro) [inline README.md:146]: Please link to this page for maven setup: https://backbase.atlassian.net/wiki/spaces/BAT/pages/270696566/Maven+configuration.

File: README.md [hunk 3]
```diff
@@ -139,6 +139,18 @@ bb-fuel supports multiple DBS versions. See below which version maps to the requ
...
+    - You may also install `maven` by using `homebrew` with following command: `brew install maven`.
+- Clone the [BBFuel](https://github.com/Backbase/bb-fuel) repository.
```
Comment (lehphyro) [inline README.md:148]: Replace "the BBFuel" with "this"

File: README.md [hunk 4]
```diff
@@ -139,6 +139,18 @@ bb-fuel supports multiple DBS versions. See below which version maps to the requ
...
+- Link Backbase artifacts to your local project by following this [guide](https://backbase.atlassian.net/wiki/spaces/NMB/pages/1819150080/How+to+configure+BB-Fuel+for+Backbase+artifacts).
```
Comment (lehphyro) [inline README.md:149]: The previous maven setup link I shared is the "official" one.

File: README.md [hunk 5]
```diff
@@ -139,6 +139,18 @@ bb-fuel supports multiple DBS versions. See below which version maps to the requ
...
+- Open the project in IntelliJ
```
Comment (vinibackbase) [inline README.md:150]: IntelliJ is not required

File: README.md (hunk 2 above)
Comment (richard-blank) [reply README.md:146]: Please keep in mind there are external developers as well. This maven setup is accessible for our customers:
https://community.backbase.com/documentation/DBS/latest/backbase_ref_maven_configuration

**Pull request closed without merging**